src/storage/src/hummock/shared_buffer/mod.rs

```diff
@@ -233,7 +238,7 @@ impl SharedBuffer {
                 UncommittedData::Batch(batch) => {
                     range_overlap(key_range, batch.start_user_key(), batch.end_user_key())
```

Comment:
We may also check whether the batch.table_id matches the passed table_id

Reply:
> We may also check whether the batch.table_id matches the passed table_id

I think that range overlap for batch indicates that table_id match